docs/guide/reporters.md

```diff
@@ -0,0 +1,428 @@
+---
+title: Reporters | Guide
+---
+
+# Reporters
+
+Vitest provides several built-in reporters to display test output in different formats, as well as the ability to use custom reporters. You can select different reporters either by using the `--reporter` command line option, or by including a `reporters` property in your [configuration file](https://vitest.dev/config/#reporters). If no reporter is specified, Vitest will use the `default` reporter as described below.
+
+Using reporters via command line:
+
+```bash
+npx vitest --reporter=verbose
+```
+
+Using reporters via `vite.config.ts`:
+
+```ts
+/// <reference types="vitest" />
+import { defineConfig } from 'vite'
+
+export default defineConfig({
+  test: {
+    reporters: ['verbose']
+  },
+})
+```
+
+## Reporter Output
+
+By default, Vitest's reporters will print their output to the terminal. When using the `json`, `html` or `junit` reporters, you can instead write your tests' output to a file by including an `outputFile` [configuration option](https://vitest.dev/config/#outputfile) either in your Vite configuration file or via CLI.
+
+Command line:
+```bash
+npx vitest --reporter=json --outputFile=./test-output.json
+```
+
+`vite.config.ts`
+```ts
+export default defineConfig({
+  test: {
+    reporters: ['json'],
+    outputFile: './test-output.json'
+  },
+})
+```
+
+## Combining Reporters
+
+You can use multiple reporters simultaneously to print your tests results in different formats. For example:
+
+```bash
+npx vitest --reporter=json --reporter=default
+```
+
+```ts
+export default defineConfig({
+  test: {
+    reporters: ['json', 'default'],
+    outputFile: './test-output.json'
+  },
+})
+```
+
+The above example will both print the test results to the terminal in the default style, and write them as JSON to the designated output file.
+
+When using multiple reporters, it's also possible to designate multiple output files, as follows:
+
+```ts
+export default defineConfig({
+  reporters: ['junit', 'json', 'verbose'],
+  outputFile: {
+    junit: './junit-report.xml',
+    json: './json-report.json',
+  },
+})
+```
+
+This example will write separate JSON and XML reports as well as printing a verbose report to the terminal.
+
+## Built-in Reporters
+
+### Default reporter
+
+By default (i.e. if no reporter is specified), Vitest will display results for each test suite hierarchically as they run, and then collapse after a suite passes. When all tests have finished running, the final terminal output will display a summary of results and details of any failed tests.
+
+```bash
```

Comment:
do we need an example for the default reporter? I don't think there's a need to pass --reporter=default. maybe the output example is enough

Reply:
Agreed, I don't think it's necessary. Deleted the terminal and config examples